File: problems/HackerRank/2521_exercises/tree/invertTree.c
struct TreeNode *invertTree(struct TreeNode *root)
{

	if (!root)
	{
		return root;
	}

	struct TreeNode *temp = root->left;
	root->left = root->right;
	root->right = temp;

	invertTree(root->left);
	invertTree(root->right);

	return root;
}

File: problems/LeetCode/tree/invertTree.py
def invertTree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
		if not root: return None

		
		# temp = root.left
		# root.left = root.right
		# root.right = temp

		root.left, root.right = root.right, root.left

		self.invertTree(root.left)
		self.invertTree(root.right)

		return root